cli/src/api/build.ts

```diff
@@ -150,7 +150,8 @@ class Builder {
     const controller = new AbortController()
 
     const buildTask = new Promise<void>((resolve, reject) => {
-      const buildProcess = spawn('cargo', this.args, {
+      const command = this.options.useCross ? 'cross' : 'cargo'
```

Comment:
I'd like to add an extra environment variable `CARGO` to control the `cargo` command like this:

```ts
process.env.CARGO ?? (this.options.useCross ? 'cross' : 'cargo')
```

Reply:
> I'd like to add an extra environment variable `CARGO` to control the `cargo` command like this:
> 
> ```ts
> process.env.CARGO ?? (this.options.useCross ? 'cross' : 'cargo')
> ```

🤔 For what purpose?